index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TanmayiPortfolio</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+    <!--Outer div consists toolbar and headline-->
+    <div class="wrapper">
+        <!--section one toolbar-->
+        <header class="toolbar">
+          <div class="statusbar">
+            <div class="time">
+                <img src="./media/TimeStyle.svg" alt="Time stamp">
+            </div>
+            <div class="bars">
+                <img src="./media/CellularConnection.svg" alt="signal bar">
+                <img src="./media/Wifi.svg" alt="wifi bar">
+                <img src="./media/Battery.svg" alt="battery">
+            </div>
+          </div>
+          <nav class="logo-hamburger">
+            <a href='#' class="logo"><span>T</span></a>
+            <a href='#' class="hamburger"><img src="./media/hamburger.png" alt="menu"></a>
+          </nav>
+        </header>
+        <!--section two headline-->
+        <section class="headline">
+          <div class="primary text">
+            <p>Hey there.<br>
+                I'm Tanmayi.<br>
+            </p>
+            <p id="green">I'm a software developer</p>
+          </div>
+          <div class="secondary text">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hesitate to contact me.</p>
+          </div>
+          <!--div for social links-->
+          <div class="socials">
+            <a class='social-item' href='https://twitter.com/TanmayiManku' target="_blank" rel="noopener"><img src="./media/twitter.svg" alt="twitter"></a>
+            <a class='social-item' href='https://www.linkedin.com/in/tanmayi-manku-99195720a/' target="_blank" rel="noopener"><img src="./media/Linkedinicon.svg" alt="linkedin"></a>
+            <a class='social-item' href='https://medium.com/' target="_blank" rel="noopener"><img src="./media/medium.svg" alt="medium"></a>
+            <a class='social-item' href='https://github.com/tan12082001' target="_blank" rel="noopener"><img src="./media/github.svg" alt="github"></a>
+            <a class='social-item' href='#' rel="noopener"><img src="./media/angellist.svg" alt="angellist"></a>
+          </div>
```

Comment:
:red_circle:  I think You can also use the ul and li tags to structure your menu items. This is for semantic reasons, as it helps search engines understand the structure of your website.

The ul tag represents an unordered list, and the li tag represents a list item. By using these tags, you are telling search engines that your menu items are a list of related items. This can help your website rank higher in search results for queries related to your menu items.

Your content should be similar to this :arrow_down:
  ```
 <div class="socials">
     <ul>
            <li><a class='social-item' href='https://twitter.com/TanmayiManku' target="_blank" rel="noopener"><img src="./media/twitter.svg" alt="twitter"></a></li>
           .....
           <li>...............</li>
    </ul>
</div>
```